ocaml/typing/typetexp.ml

```diff
@@ -709,9 +709,13 @@ and transl_type_aux env ~row_context ~aliased ~policy mode styp =
           let l = transl_label l (Some arg) in
           let arg_cty =
             if Btype.is_position l then
+              (* XXX jrodri for goldfirere: Hmm, I think that I do not understand the
```

Comment:
Don't worry about it -- not worth trying to recreate whatever simplification I thought might be possible last summer. There are better ways to spend our time. :)

Reply:
Cool that sounds reasonable. I've removed the cr question.